packages/gui/src/components/nfts/NFTPreview.tsx

```diff
@@ -162,22 +420,268 @@ export default function NFTPreview(props: NFTPreviewProps) {
         </body>
       </html>,
     );
-  }, [file, statusText, isStatusError]);
+  }, [file, statusText, isStatusError, thumbnail, error]);
+
+  function mimeType() {
+    const pathName: string = new URL(file).pathname;
+    return mime.lookup(pathName);
```

Comment:
Several of my NFTs don't have a `pathName` with a file extension. For those NFTs, this function returns `false` which then causes exceptions when trying to call `<false>.match(/whatever/)`.

As a workaround, I modified this function to return `undefined` if lookup returned `false`, and then updated all occurrences of `mimeType().` to `mimeType()?.`

That got me past the exceptions so that the gallery could render.

Reply:
Original exception:

```
Uncaught TypeError: mimeType(...).match is not a function
    at NFTPreview.tsx:394:27
    at updateMemo (react-dom.development.js:15867:1)
    at Object.useMemo (react-dom.development.js:16413:1)
    at useMemo (react.development.js:1532:1)
    at NFTPreview (NFTPreview.tsx:310:25)
    at renderWithHooks (react-dom.development.js:14985:1)
    at updateFunctionComponent (react-dom.development.js:17356:1)
    at beginWork (react-dom.development.js:19063:1)
    at HTMLUnknownElement.callCallback (react-dom.development.js:3945:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:3994:1)```